app/views/lateness/_picker.html.erb

```diff
@@ -1,150 +1,109 @@
-<div id="<%= f_late.object_name %>_lateness-configuration" class="">
+<div class="form-tabs-pane" data-type="<%= f.object.lateness_config&.type %>">
   <% if !within_course %>
   <p class="col-sm-12">(This is the default late policy for all
     assignments in this course.)</p>
   <% end %>
-  <%= f_late.hidden_field :type %>
+
   <ul class="nav nav-tabs">
     <% if within_course %>
-    <li><a data-toggle="tab" href="#<%= f_late.object_name %>_UseCourseDefaultConfig">Course default</a></li>
+    <li><a href="#" class="lateness-tab" data-type="default">Course default</a></li>
+    <% end %>
+    <li><a href="#" class="lateness-tab" data-type="LatePerDayConfig">Late-per-day</a></li>
+    <li><a href="#" class="lateness-tab" data-type="LatePerHourConfig">Late-per-hour</a></li>
+    <li><a href="#" class="lateness-tab" data-type="FixedDaysConfig">Limited number</a></li>
+    <% if within_course %>
+    <li><a href="#" class="lateness-tab" data-type="copy">Use existing settings</a></li>
     <% end %>
-    <li><a data-toggle="tab" href="#<%= f_late.object_name %>_LatePerDayConfig">Late-per-day</a></li>
-    <li><a data-toggle="tab" href="#<%= f_late.object_name %>_LatePerHourConfig">Late-per-hour</a></li>
-    <li><a data-toggle="tab" href="#<%= f_late.object_name %>_FixedDaysConfig">Limited number</a></li>
-    <li><a data-toggle="tab" href="#<%= f_late.object_name %>_reuse-lateness">Copy existing settings</a></li>
   </ul>
-  <div class="tab-content">
+  <div class="form-tabs-content">
     <% if within_course %>
-    <div id="<%= f_late.object_name %>_UseCourseDefaultConfig" class="tab-pane equal-height-group form-group">
-      <p>Use course-default lateness policy:</p>
-      <p><%= @course.lateness_config.to_s %></p>
-    </div>
+      <div v-if="type == 'default'">
+        <%= f.hidden_field :lateness_config_id %>
+        <div class="tab-pane equal-height-group lateness-pane form-group">
+          <p>Use course-default lateness policy:</p>
+          <p><%= @course.lateness_config.to_s %></p>
+        </div>
+      </div>
     <% end %>
 
-    <%= f_late.fields_for :LatePerDayConfig do |lpd| %>
-    <div id="<%= f_late.object_name %>_LatePerDayConfig" class="tab-pane equal-height-group form-group">
-      <div class="form-inline">
-        Deduct
-        <%= lpd.spinner "percent_off", 20, :min => 0, :max => 100, :delta => 5 %>
-        percent every
-        <%= lpd.spinner "frequency", 1, :min => 0, :max => 100 %>
-        days late to a maximum of
-        <%= lpd.spinner "max_penalty", 100, :min => 0, :max => 100, :delta => 5 %>
-        percent off.
-      </div>
-      <br>
-      <div class="form-inline">
-        Stop accepting assignments after
-        <%= lpd.spinner "days_per_assignment", 5, :min => 0, :max => 5 %>
-        days.
-      </div>
+    <div v-if="type == 'LatePerDayConfig'">
```

Comment:
v-if is a Vue.js directive, saying that this div only exists if the conditional is true, with "type" referring to the type field of the Vue model.